modules/jwplayerVideoProvider.js

```diff
@@ -0,0 +1,997 @@
+import {
```

Comment:
These imports suffer from the same problem, but they should work because they are immutable. But if you look at the build output, you should see them duplicated, which is still not ideal.

Reply:
Not sure I understand. What is wrong with 
```
import {
  PROTOCOLS, API_FRAMEWORKS, VIDEO_MIME_TYPE, PLAYBACK_METHODS, PLACEMENT, VPAID_MIME_TYPE
} from './videoModule/constants/ortb.js';
``` 
?